src/components/BackArrow.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 
 const BackArrow = ({onClick}) => {
     return (
-        <div className="BackArrow" onClick={onClick} >
+      <div className="BackArrow" onClick={onClick} >
```

Comment:
make sense to do it as a button (the same for NextArrow)

Reply:
BUGS FIXED IN PULL REQUEST LESSON-4